app/paypal.py

```
@@ -0,0 +1,241 @@
+                        if item['item_quantity'] > 0:
+                            sql_connection.ping()
+                            if item['item_code'] == PAYPAL_ITEMCODE_100MESSAGES:
+                                log_info('FOUND 100MESSAGES in transaction')
+                                sql_connection.cursor().execute("INSERT INTO purchases (user_id, at_datetime, amount, product, paypal_transaction_id) VALUES (?, ? ,?, ?, ?)", [
+                                    user_id,
+                                    transaction['transaction_info']['transaction_initiation_date'],
+                                    item['item_quantity'] or 1,
+                                    '100MESSAGES',
+                                    transaction['transaction_info']['transaction_id']
+                                ])
+                                for i in range(item['item_quantity']):
```

Ruff (0.12.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)